zephyr/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+if(CONFIG_LIBLC3CODEC)
+
+zephyr_library_named(liblc3codec)
+zephyr_library_compile_options(-std=c++11 )
+
+zephyr_include_directories(../Api)
+zephyr_include_directories(../Common)
+zephyr_include_directories(../Common/KissFft)
+zephyr_include_directories(../Common/Tables)
+zephyr_include_directories(../TestSupport)
+zephyr_include_directories(../zephyr)
+
+zephyr_library_sources(            ../Common/DctIV.cpp)
```

Comment:
Github shows a lot of white space between the "(" and the "../" - is there a reason for the white space?

Reply:
Only reason being that it was written like that in another module - I'll delete.